src/labels/CustomLabels.labels

```diff
@@ -2827,6 +2842,13 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Message preventing user from processing batch if there are errors.</shortDescription>
         <value>There are errors on one or more entries. You must resolve all errors before you can process this batch.</value>
     </labels>
+    <labels>
+        <fullName>bgeGridErrorConflictingGifts</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Message preventing user from processing batch if there are conflicting gifts.</shortDescription>
+        <value>There are conflicting gifts for this batch. These conflicts must be resolved before you can process the batch.</value>
```

Comment:
Recommend to revise to:
There are conflicting gifts in this batch. You will need to resolve these conflicts before you're able to process this batch.